torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3163,15 +3172,7 @@ def method_tests():
         ('scatter_add', (), (0, torch.tensor(0, dtype=torch.int64), ()), 'scalar_all_dim0', (), [0]),
         ('scatter_add', (M, S), (0, gather_variable((S, S), 1, M), (S, S)), 'alert_nondeterministic', (), [0],
             [expectedAlertNondeterministic('scatter_add_cuda_kernel', 'cuda')]),
-        ('masked_fill', (M, M), (torch.BoolTensor(M, M).bernoulli_(), 10)),
-        ('masked_fill', (M, M), (torch.BoolTensor(M, M).bernoulli_(), ()), 'tensor'),
         ('masked_fill', (M,), (torch.BoolTensor(M, M).bernoulli_(), 10), 'broadcast_lhs'),
```

Comment:
Keeping this entry here.
Could not add this entry to OpInfo as it failed (input is broadcasted so inplace variant errors out).

https://github.com/pytorch/pytorch/blob/3cf78395cbc32fa9c83b585c9ec63f960b32d17f/test/test_ops.py#L244-L250

Had to do similar for `masked_scatter`. 

@mruberry Do you think we should add a flag in `OpInfo` called `is_broadcastable` and catch the error the if it is inplace variant and input was broadcasted?

Reply:
similar to #50747. In the case of `torch.addr`, we still added these test cases but skipped the failing tests. I think it's a nice idea to add the flag, however we still want `test_variant_consistency_eager` to run for these test inputs, so we probably wanna update those test functions to handle broadcasting based on the flag.

Reply:
I have something like this in mind

```python
if is_broadcastable and check_if_any_input_broadcasted(sample.input):
     with self.assertRaises(RuntimeError):
          variant(sample)
else:
      # existing code.
```

Reply:
what will `check_if_any_input_broadcasted` do?

Reply:
I think it will be `check_if_first_input_broadcasted`

```python
def check_if_first_input_broadcasted(sample.input, expected_forward):
      return sample.input[0].shape != expected_forward.shape
```

It makes the assumption that first input will be `self` (the tensor on which inplace variant will be called).